official/transformer/v2/transformer_benchmark.py

```diff
@@ -142,6 +142,7 @@ def benchmark_1_gpu(self):
     FLAGS.batch_size = 4096
     FLAGS.train_steps = 200000
     FLAGS.steps_between_evals = 5000
+    FLAGS.static_batch = True
```

Comment:
Make a new test called   `benchmark_1_gpu_static`  so we can track the difference.  legacy graph does not seem to have this issue or not as bad running the same keras code.  

Once they both converge or we are sure of the standard we can drop one of them.  I am not certain at this time what is "right".  I don't want to run both to convergence, but for now both seems correct.